### PR TITLE
Create automatic-logo-square.svg and automatic-logo.svg

### PR DESCRIPTION
**Pull request by norwd**
This is my attempt at creating automatic dark-mode SVG as suggested in #2

This is how it looks in Light Mode:

image

And this is how it looks in Dark Mode:

image

And this is how the non-square version looks in Light Mode:

image

And lastly, in Dark Mode:

image

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -203,6 +203,19 @@ fn index() -> Node {
...
+            <li>
+                <a href="automatic-logo.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
+                    automatic-logo.svg
+                </a>
+            </li>
+            <li>
+                <a href="automatic-logo-square.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
+                    automatic-logo-square.svg
+                </a>
+            </li>
```
Comment (norwd) [inline src/main.rs]: This is how this new section renders:

image

I'm not sold on the text in parenthesis, I'd be ok just saying "White, Black, Automatic" but I don't know if that's clear enough?

**Comment by 0atman** [comment]
Fantastic! I'll do some testing and get this merged in. Step 1 is auto-test the html...

**Comment by 0atman** [comment]
Apologies @norwd, my wierd statically-compiled html library requires `"quoting"` of any strings that use punctuation that the parser can't yet figure out. - throw some quotes around the offending line to make that pass! :grinning:

**Comment by 0atman** [comment]
(why did I use this annoying way of building the site? great question, I'm a weirdo who adores compile-time verification! `html-node` already catches all invalid html (and understands all common tags' valid properties), but sadly it currently is overly strict about quoting strings. I hope to find a way to solve both soon!)

File: src/main.rs (hunk 1 above)
Comment (norwd) [reply src/main.rs]:
```suggestion

            <h3 class="text-l"><b>Automatic</b></h3>
            <ul class="list-disc">
            <li>
                <a href="automatic-logo.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
                    automatic-logo.svg
                </a>
            </li>
            <li>
                <a href="automatic-logo-square.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
                    automatic-logo-square.svg
                </a>
            </li>
```

**Comment by norwd** [comment]
@0atman, I've fixed the HTML quoting, if you approve running the workflow it should pass now 🚀

**Comment by 0atman** [comment]
Thank you! looks like a tag's missing? THIS IS WHY I SET UP THIS SYSTEM! VINDICATION! :tada: :laughing:

**Comment by norwd** [comment]
@0atman, ohh nice one, I've fixed that too now. Hopefully that's the last one 🤞

**Comment by 0atman** [comment]
Looks great, merged and pushed, thank you!

**Pull request merged**